clients/cloud/scala/README.md

```diff
@@ -0,0 +1,108 @@
+# Scala Producer and Consumer for Confluent Cloud
+
+This directory includes projects demonstrating how to use the Scala producer and consumer with Confluent Cloud.
+
+For more information, please see the [application development documentation](https://docs.confluent.io/current/api-javadoc.html)
+
+
+# Quickstart
+
+Before running the examples, **you should setup the Confluent Cloud CLI** by running the [quickstart](https://docs.confluent.io/current/quickstart/cloud-quickstart.html#cloud-quickstart).
+
+
+1. Create a topic called `testtopic`:
+
+	```shell
+	# Create page_visits topic
+	$ ccloud topic create testtopic
```

Comment:
The other examples are using a 1-partition topic.

Reply:
done